interactive/templates/interactive/new_analysis_request.html

```diff
@@ -1,26 +1,62 @@
 {% extends "base.html" %}
 
-{% block content %}
+{% block meta_title %}Request an analysis | OpenSAFELY Interactive{% endblock meta_title %}
+{% block html_class %}h-full bg-slate-50{% endblock html_class %}
+{% block body_class %}h-full{% endblock body_class %}
 
-<div class="mt-8 sm:mx-auto sm:w-full sm:max-w-2xl">
-  <h2 class="text-xl font-extrabold text-slate-900">Request an analysis</h2>
-  <div class="bg-white py-4 px-4 shadow sm:rounded-lg sm:px-10">
+{% block content %}
+<div class="flex flex-col justify-center py-12 sm:px-6 lg:px-8">
+  <div class="flex flex-col mb-6 sm:mx-auto">
+    <h1 class="text-center text-3xl font-extrabold text-slate-900 sm:w-full sm:max-w-lg mx-auto">
+      Request an analysis
+    </h1>
+  </div>
 
-    <form  class="space-y-6" method="post">
+  <div class="bg-white sm:mx-auto sm:w-full sm:max-w-xl py-8 px-4 shadow sm:rounded-lg sm:px-10">
+    <form class="space-y-6" method="post">
       {% csrf_token %}
       {{ form.non_field_errors }}
+
       <div>
-        {% include "components/form-field.html" with id=form.title.id_for_label label=form.title.label name="title" required=True type="text" %}
+        {% include "components/form-field.html" with id=form.title.id_for_label label=form.title.label name="title" required=True type="text" hint="For example: Systolic code usage" %}
+      </div>
+
+      <div x-title="codelist-select" x-data="{
+          codelist: '',
+          get codelistURL() {
+            return `https://www.opencodelists.org/codelist/${this.codelist}`
+          }
+        }">
+        {% include "components/form-field-select.html" with id=form.codelist.id_for_label label=form.codelist.label name="codelist" required=True choices=form.codelist.field.choices hint="Codelist selection have been published by OpenSAFELY on OpenCodelists and use the SNOMED CT coding system" x_model="codelist" %}
```

Comment:
Should this be "Codelist selection **has** been published..."?

Reply:
Good point, I'll tweak that wording.

> Codelists are published by OpenSAFELY on OpenCodelists and use the SNOMED CT coding system